test/forms/ContactPersonEmailAddressFormProviderSpec.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.behaviours.StringFieldBehaviours
+import play.api.data.FormError
+
+class ContactPersonEmailAddressFormProviderSpec extends StringFieldBehaviours {
+
+  val form = new ContactPersonEmailAddressFormProvider()()
+
+  val field = "contact-email-address"
+
+  ".contact-email-address" - {
+    behave like mandatoryField(
+      form,
+      field,
+      requiredError = FormError(field, "contactPersonEmailAddress.error.required")
+    )
+  }
```

Comment:
Can you add a test to check that the email format validation works for this particular form provider?